src/Api.ts

```diff
@@ -3,46 +3,40 @@ import { ChallengeInput } from "./models/ChallengeInput"
 import { User } from "./models/User"
 
 
-function postRequest(url: string, bodyObject: {}) {
-    return fetch(url, {
+async function postRequest<T>(url: string, bodyObject: {}) {
+    const res = await fetch(API.BASEURL + url, {
         method: "POST",
         headers: {
             "Content-Type": "application/json",
             "Access-Control-Allow-Origin": "http://localhost:3000"
         },
         mode: "cors",
-        credentials: "include",
+        // credentials: "include",
```

Comment:
comment doet niks en voegd niks toe

Reply:
hebben we later nog nodig waarschijnlijk

Reply:
voegt*